org.eclipse.xtext.builder.standalone.tests/src-gen/org/eclipse/xtext/builder/tests/builderTestLanguage/impl/BuilderTestLanguagePackageImpl.java

```diff
@@ -72,7 +72,6 @@ private BuilderTestLanguagePackageImpl()
   {
     super(eNS_URI, BuilderTestLanguageFactory.eINSTANCE);
   }
-
```

Comment:
is these changes cause you dont use emf 2022-03 as tp?

Reply:
@cdietrich it can be: I'm always using the latest TP. Do you want me to remove them from the PR?

Reply:
this is an "argggggggg" problem. strictly i like to have the variant with the minimal emf version.
so yes. but i can also live with it.
am not sure if we will run into a new emf version where there is a real change

Reply:
@cdietrich I've pushed force without the modified Java files